web-client/src/presenter/computeds/Dashboard/externalUserCasesHelper.test.ts

```diff
@@ -18,56 +18,79 @@ describe('externalUserCasesHelper', () => {
   beforeEach(() => {
     baseState = {
       closedCases: [
-        { docketNumber: '101-20' },
-        { docketNumber: '102-20' },
+        {
```

Comment:
Working on updating the case setup with `caseCaption` explicitly set and checking for the `caseTitle`.

Reply:
Done. [627430e](https://github.com/ustaxcourt/ef-cms/pull/4081/commits/627430e3e8dbb075d941a9e232abd6c9712a008c)